src/main/proto/response_code.proto

```diff
@@ -1087,4 +1087,9 @@ enum ResponseCodeEnum {
    * confirm the hash of the file to be used in the upgrade.
    */
   UPDATE_FILE_HASH_DOES_NOT_MATCH_PREPARED = 278;
+
+  /**
+   * Evm tx processor should confirm that provided storage key is not bigger than 32 bytes
```

Comment:
I suggest not to mention the EvmTxProcessor directly and just explain that this response code is related to a precheck for storage keys and they should not be larger than 32 bytes